src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int MAX_CAPACITY = 10;
+    private Object[] array = new Object[MAX_CAPACITY];
+    private int size;
+
+    public void grow() {
```

Comment:
It's our internal logic, no need to expose it.
Please also move it after public methods after modificator is changed
```suggestion
    private void grow() {
```